client/lib/redux-bridge/index.js

```diff
@@ -0,0 +1,15 @@
+let reduxStore = null;
+
+export function setReduxStore( store ) {
+	reduxStore = store;
+}
+
+/**
+ * Dispatch an action against the current redux store
+ */
+export function reduxDispatch( ...args ) {
```

Comment:
maybe just expose the getState function of the store instead of the whole store 🤔.

This is all so ripe for abuse! Lets reduxify more things!